### ISSUE TITLE
Supervised Learning - Non-linear deformations

### ISSUE THREAD

**Issue by finalelement** [Project-MONAI/MONAILabel#237]
Non-linear deformation of the image

**Comment by finalelement** [comment]
These are first being implemented in MONAI Core, before we start importing them for use.

**Comment by SachidanandAlle** [comment]
I guess we can close this issue.. we may not needed now.
